pom.xml

```
@@ -0,0 +1,298 @@
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
+        <maven.bundle.plugin.version>2.4.0</maven.bundle.plugin.version>
+        <maven.buildnumber.plugin.version>1.4</maven.buildnumber.plugin.version>
+        <maven.compiler.plugin.encoding>UTF-8</maven.compiler.plugin.encoding>
+        <maven.compiler.plugin.source>1.8</maven.compiler.plugin.source>
+        <maven.compiler.plugin.target>1.8</maven.compiler.plugin.target>
+        <maven.compiler.plugin.version>2.3.1</maven.compiler.plugin.version>
+        <maven.surefire.plugin.version>2.22.0</maven.surefire.plugin.version>
+        <maven.scm.version>1.7.0.wso2v3</maven.scm.version>
+        <maven.war.plugin.version>3.2.2</maven.war.plugin.version>
+
+        <!-- check-style plugin properties -->
+        <maven.checkstyleplugin.excludes>**/gen/**/*</maven.checkstyleplugin.excludes>
+
+        <!-- findbugs plugin properties -->
+        <mavan.findbugsplugin.exclude.file>findbugs-exclude-filter.xml</mavan.findbugsplugin.exclude.file>
+        <maven.findbugsplugin.version>3.0.5</maven.findbugsplugin.version>
+
+        <commons.beanutils.version>1.9.4</commons.beanutils.version>
+        <cxf-bundle.version>3.2.8</cxf-bundle.version>
+        <com.fasterxml.jackson.jaxrs-json-provider-version>2.9.7</com.fasterxml.jackson.jaxrs-json-provider-version>
+        <com.fasterxml.jackson.version>2.9.7</com.fasterxml.jackson.version>
+        <com.fasterxml.jackson.annotation.version>2.9.8</com.fasterxml.jackson.annotation.version>
+        <jackson-databind.version>2.9.9.3</jackson-databind.version>
+        <jackson-jaxrs-json-provider.version>2.9.9</jackson-jaxrs-json-provider.version>
+        <jackson.version>1.9.13</jackson.version>
+        <javax.ws.rs-api.version>2.1.1</javax.ws.rs-api.version>
+        <swagger-jaxrs.version>1.5.20</swagger-jaxrs.version>
+        <org.apache.felix.annotations.version>1.2.4</org.apache.felix.annotations.version>
+        <org.apache.cxf.version>3.3.1</org.apache.cxf.version>
+        <springframework.version>4.1.6.RELEASE</springframework.version>
+
+        <!-- open-api tool properties -->
+        <openapi.tools.version>4.1.2</openapi.tools.version>
+        <org.codehaus.mojo.version>3.2.0</org.codehaus.mojo.version>
+
+        <!-- wso2 product related properties -->
+        <identity.governance.version>1.4.1</identity.governance.version>
+        <carbon.identity.framework.version>5.16.95</carbon.identity.framework.version>
+        <identity.workflow.impl.bps.version>5.2.0</identity.workflow.impl.bps.version>
+        <identity.event.handler.version>1.2.9</identity.event.handler.version>
+
+        <!-- otp extension properties -->
+        <org.wso2.carbon.extension.identity.smsotp.common.version>3.0.15-SNAPSHOT
+        </org.wso2.carbon.extension.identity.smsotp.common.version>
+
+        <!-- Carbon kernel version -->
+        <carbon.kernel.version>4.6.0</carbon.kernel.version>
```

```suggestion
        <carbon.kernel.version>4.7.0-m2</carbon.kernel.version>
```

Change kernel version to 4.7.0-m2 since it's the latest version in product-is

---

Fixed in 133c6682327014072c3c19ad62360620a9904b91